docs/helping.md

```diff
@@ -3,3 +3,85 @@
+
+Once they hit a button to open a ticket, they will fill out the popup with information which will later be shared in their ticket as a message sent as an embed by the ticket bot. 
+This will be the first message in the channel and will look something similar to this:\
+![The embed sent in a ticket with the information filled out by the user](images/ticket-open-message.png)
+
+Note the buttons below the embed.\
+Hitting `Close` will close the ticket without a reason for closing it.\
+`Close with reason` will let you give a reason as to why you closed the ticket.\
+`Claim` allows you to "claim" a ticket and lets other helpers know that you'll be solving the ticket. 
+This doesn't usually get used for Northstar, however other helpers can still message in the channel even after it's claimed so don't worry if you accidentally claim a ticket/end up not being able to solve it.
+
+These buttons, however, are not always easy to scroll back up to, especially if the ticket has been open for a while. This is where slash commands come in.
+
+There are 3 major slash commands used for tickets. These are `/close`, `/closerequest`, and `/add`.
+
+`/close` by itself will close the ticket immediately, without a reason. Because it's a slash command however, you can give a reason for closing using the `reason` option when typing the slash command.
+
+`/add` will add a normal (non ticket-viewing) user to a ticket. This is useful if, for example, someone who created a mod and can't see the tickets has one of their mods cause an error for a user and you can't diagnose it. This is also useful for cases where someone opens a ticket for someone else.
+
+You may have noticed that `/closerequest` has been skipped. That's because there's sort of a "etiquette" surrounding it.
+
+Generally, you _should_ use `/closerequest` for closing a ticket. There's much less chance the end user gets upset, and it can be denied last minute if another issue comes up.\
+![What using the `/closerequest` command looks like](images/closerequest.png)
+
+From here, the user can accept the close request, which in turn closes the ticket, or deny the close request, while results in an edited embed telling you they denied it.
+
+To _use_ the `/closerequest` command, there are a few "ways" to use it.\
+Note that `close_delay` is a number, meaning the time in hours before the ticket autocloses.\
+`reason` is a box to input the reason for a close request.\
+Using "ac" as described later lets other people know how long until the ticket will close ("ac" meaning **A**uto **C**lose). This helps yourself and others when looking at tickets because the tickets bot doesn't show when a ticket is set to auto close.
+
+If the ticket seems resolved/the user has said they have no issues, use `/closerequest` with `close_delay` set to `8` and the `reason` set to "resolved, ac 8hr"
```

Comment:
I'd say more like 12 hours to give them a bit more time to come back if need be.

```suggestion
If the ticket seems resolved/the user has said they have no issues, use `/closerequest` with `close_delay` set to `12` and the `reason` set to "resolved, ac 12hr"
```